Client/src/components/contanier/staff/Courses/ViewCourse-Details/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import "./index.css";
+import { Redirect } from 'react-router-dom';
+import Table from "../../../../common/Table";
+import Loading from '../../../../Layout/Loading'
+
+export default class ViewCourseDetails extends Component {
+  state = {
+    headings: null,
+    rows: null,
+    Error: null
+  }
+  componentDidMount() {
+    const { id } = this.props.match.params;
+    if (id) {
```

Comment:
@Fatmasiam I think you should always have an `id` here or this component wouldn't even be on screen, because of the route path, so you won't have to worry about checking `if (id)`.